pkg/auth/service.go

```diff
@@ -1147,10 +1155,21 @@ func NewAPIAuthServiceWithClient(client ClientWithResponsesInterface, externalPr
 	}, nil
 }
 
-func CheckPermissions(ctx context.Context, node permissions.Node, username string, policies []*model.Policy) CheckResult {
+func (n *MissingPermissions) String() string {
+	if len(n.Denied) != 0 {
+		return fmt.Sprintf("denied permission to %s", strings.Join(n.Denied, ","))
+	}
+	if len(n.Unauthorized) != 0 {
+		return fmt.Sprintf("missing permission to %s", strings.Join(n.Unauthorized, ","))
```

Comment:
l.935 will cause a message that says `"insufficient permissions: missing permission to ..."`.  That's a bit wordy, so perhaps
```suggestion
		return fmt.Sprintf("not allowed to %s", strings.Join(n.Unauthorized, ","))
```
An added benefit is that it uses the same word "allow" from the policy language.